### PR TITLE
Olwa pr

### PR DESCRIPTION
**Pull request by oliver-walker-hms**

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Flexy OSIsoft Changelog
 
+## V2.8
+### Major Changes
++ None
+### Minor Changes
+Fixed malformed post requests for piwebapi calls
+Updated libraries
```
Comment (TomKimsey) [inline docs/CHANGELOG.md:8]: Missing bullets

**Review by TomKimsey — CHANGES_REQUESTED** [review]

File: docs/CHANGELOG.md (hunk 1 above)
Comment (oliver-walker-hms) [reply docs/CHANGELOG.md]: fixed

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 # Flexy OSIsoft Changelog
 
+## V2.8
+### Major Changes
++ None
+### Minor Changes
++ Fixed malformed post requests for piwebapi calls
```
Comment (TomKimsey) [inline docs/CHANGELOG.md:7]: Weren't there GETs and PUTs that were fixed as well?

Comment (oliver-walker-hms) [reply docs/CHANGELOG.md]: fixed

**Review by TomKimsey — APPROVED** [review]

File: docs/CHANGELOG.md (hunk 2 above)
Comment (alexjhawk) [reply docs/CHANGELOG.md]: The GETs and PUTs change was not performed in a library. IT was done within the OSISoft connector at https://github.com/hms-networks/flexy-osisoft-connector/pull/91/files

**Review by alexjhawk — CHANGES_REQUESTED** [review]
See comment: https://github.com/hms-networks/flexy-osisoft-connector/pull/92#discussion_r573126718

"The GETs and PUTs change was not performed in a library. IT was done within the OSISoft connector at https://github.com/hms-networks/flexy-osisoft-connector/pull/91/files"

File: docs/CHANGELOG.md (hunk 2 above)
Comment (oliver-walker-hms) [reply docs/CHANGELOG.md]: The library update is a separate change from a few weeks ago

Comment (TomKimsey) [reply docs/CHANGELOG.md]: What are you referring to @alexjhawk?

Comment (alexjhawk) [reply docs/CHANGELOG.md]: @TomKimsey was your original comment not referring to the GETs and PUTs that were changed in https://github.com/hms-networks/flexy-osisoft-connector/pull/91/files?

Comment (TomKimsey) [reply docs/CHANGELOG.md]: I never implied it was performed in a library.

**Review by alexjhawk — APPROVED** [review]
I must've confused myself. I thought that the "Updated libraries" was added to the changelog as a result of @TomKimsey's comment about GETs and PUTs.

**Pull request merged**